app/views/tasks/show.html.haml

```diff
@@ -54,6 +54,18 @@
             %td.govuk-table__cell.govuk-table__cell--numeric
               = pluralize(@submission.order_count, 'row')
 
+    %h2.govuk-heading-m
```

Comment:
Do we actually want to make the correction process publicly accessible whilst it's only partially implemented? The user experience might be confusing: users that click "Correct this return" for a no-business return will hit a dead-end (the correction page will not have any buttons), and users wanting to correct a business return with another business return will only have the option to replace with a no-business return. We might be better off omitting the link to the correction process until the rest of the pieces are in place. We could still share the URL directly with any users that contact support wanting to replace a business with no-business.

Reply:
Feature flag or remove the code?

Reply:
Do you have a preference? I'd say go with the easy option for now, which is probably just don't include he link.